Cargo.toml

```diff
@@ -44,11 +44,13 @@ sha1 = { version = "0.10", default-features = false }
 sha2 = "0.10"
 num-derive = "0.4"
 num-traits = { version = "0.2", default-features = false }
-picky = { version = "7.0.0-rc.12", default-features = false }
-picky-asn1 = "0.10"
-picky-asn1-der = "0.5"
-picky-asn1-x509 = "0.14"
-picky-krb = "0.9"
+
+picky = { path = "../picky-rs/picky", default-features = false }
+picky-asn1 = { path = "../picky-rs/picky-asn1" }
+picky-asn1-der = { path = "../picky-rs/picky-asn1-der" }
+picky-asn1-x509 = { path = "../picky-rs/picky-asn1-x509" }
+picky-krb = { path = "../picky-rs/picky-krb" }
```

Comment:
https://github.com/Devolutions/picky-rs/pull/369

Reply:
Published!

Reply:
@CBenoit I don't see a new version of the `picky-krb` on crates.io: https://crates.io/crates/picky-krb/versions

Reply:
Oops… Looks like a bug, but it got moving again now